src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -32,21 +37,25 @@ import {
   textStyle
 } from './ChatItemText.style';
 import {replyIcon} from './ChatItemTargetText';
+import {formatBytes} from '../../../utils/string/StringUtils';
 
 const ChatItemMyTextV2 = ({
   username = 'Anonymous User',
   time = '4h',
   isContinuous = false,
   message = '',
+  attachments = [],
   status = ChatStatus.PENDING,
   avatar,
   chatType,
   messageType,
   data
 }: ChatItemMyTextProps) => {
+  const navigation = useNavigation();
   const {
     setReplyPreview,
     onContextMenuPressed,
+    onOpenMediaPreview,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` prop is added to the component, but there's no prop validation or type checking. This could lead to unexpected behavior if the wrong data type is passed. Consider adding PropTypes or TypeScript interfaces for better type safety.



<!-- This is an auto-generated comment by OSS CodeRabbit -->